hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerRestart.java

```diff
@@ -188,12 +197,29 @@ public void testRestartOMWithKeyOperation() throws Exception {
     Assert.assertTrue(ozoneBucket.getName().equals(bucketName));
 
     String data = "random data";
-    OzoneOutputStream ozoneOutputStream = ozoneBucket.createKey(key,
+    OzoneOutputStream ozoneOutputStream1 = ozoneBucket.createKey(key1,
```

Comment:
Nit: renaming `ozoneOutputStream` seems unnecessary, but no need to revert unless further changes are requested by other reviewers.